samples/crash/build.gradle

```diff
@@ -25,6 +25,14 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
+
+    variantFilter { variant ->
+        if (variant.buildType.name == "release") {
+            // This is a sample app that we are not releasing. Save some time and do not build
+            // release versions.
+            setIgnore(true)
+        }
+    }
```

Comment:
Instead of adding this to all the module `build.gradle` files it would be nice to do that in the root `build.gradle` for **app** modules only - there's already a block that configures all modules.